packages/website/scripts/generateModulePages.mjs

```diff
@@ -0,0 +1,113 @@
+| \`description\` | Adds a description to the variable in the playground.       |
+| \`showAs\` | Change the default view for the value when displayed. |
+| \`format\` | Format a number, date, or duration when displayed. |
+| \`hide\` | Don't show the variable in the Playground |
+## Definitions
+`,
+  },
+  {
+    name: "Date",
+    intro: `---
+description: Dates are a simple date time type.
+---
+import { SquiggleEditor, FnDocumentationFromName } from "@quri/squiggle-components";
+
+# Date`,
+  },
+  {
+    name: "Duration",
+    intro: `---
+description: Durations are a simple time type, representing a length of time. They are internally stored as milliseconds, but often shown and written using seconds, minutes, hours, days, etc.
+---
+import { SquiggleEditor, FnDocumentationFromName } from "@quri/squiggle-components";
+
+# Duration
+Durations are a simple time type, representing a length of time. They are internally stored as milliseconds, but often shown and written using seconds, minutes, hours, days, etc.
+
+
+| **Unit Name** | **Example** | **Convert Number to Duration** | **Convert Duration to Number** |
+|---------------|----------------------------|--------------------------------------------|--------------------------------------------|
+| Minute        | \`5minutes\`                   | \`fromMinutes(number)\`                      | \`toMinutes(duration)\`                      |
+| Hour          | \`5hour\`                     | \`romHours(number)\`                        | \`toHours(duration)\`                        |
+| Day           | \`5days\`                      | \`fromDays(number)\`                         | \`toDays(duration)\`                         |
+| Year          | \`5years\`                     | \`fromYears(number)\`                        | \`toYears(duration)\`                        |
+
+This table now presents the information in a clear and concise manner, focusing only on the essential columns.
+`,
+  },
+];
+
+function toMarkdownDefinitions(definitions) {
+  return `\`\`\`
+  ${definitions.join("\n")}
+  \`\`\``;
+}
+
+function toMarkdown(documentation) {
+  return `### ${documentation.name}
+${documentation.description || ""}
+<FnDocumentationFromName functionName="${
+    documentation.nameSpace + "." + documentation.name
+  }" showNameAndDescription={false} size="medium" />
+`;
+}
+
+const main = async ({ name, intro }) => {
+  const namespaceNames = getAllFunctionNamesWithNamespace(name);
+  const content =
+    intro +
+    "\n" +
+    namespaceNames
+      .map((name) => {
+        return toMarkdown(getFunctionDocumentation(name));
+      })
+      .join("\n");
+  fs.writeFile(targetFilename(name), content, (err) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+    console.log(`Content written to ${targetFilename(name)}`);
+  });
+};
+
+//Remember to add any new Modules to .gitignore
+sections.forEach(main);
```

Comment:
Note that this call will process all files in parallel.

Its fine now but if we ever decide to e.g. update `_meta.json` dynamically on each generated file, it'd cause race condition bugs.

So `for (const section in sections) { await main(section); }` would be more reliable.